docker/reverseproxy/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+services:
+  reverse-proxy:
+    container_name: reverse-proxy
+    image: nginxinc/nginx-unprivileged:alpine
+    ports:
+      - 80:80
+      - 443:443
+    volumes:
+      - ./conf.d:/etc/nginx/conf.d
+    networks:
+      - production
+      - testing
+      - reverse_proxy_network
+    depends_on:
+      - acme-sh
+
+  acme-sh:
+    image: neilpang/acme.sh
+    container_name: acme-sh
+    command: daemon
+    volumes:
+      - /home/monitoring/ssl_certificates:/acme.sh
+    networks:
+      - reverse_proxy_network
+
+networks:
+  production:
+    external:
+      name: production_nginx_network
```

Comment:
Where are these networks defined? In the main `docker-compose.yml` I only see `nginx_network`.

Reply:
docker merges the folder name and the name we have in the docker file. So I assume that we start the production containers from the folder production.